docs/HowToGuides/GettingStarted.md

```diff
@@ -259,10 +260,14 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
        --sccache --release-debuginfo --swift-disable-dead-stripping --test \
        --xcode
      ```
+   Linux (uses Ninja):
+     ```sh
+     utils/build-script --release-debuginfo --test --skip-early-swift-driver
```

Comment:
What is the reason for recommending `--skip-early-swift-driver` here?

Reply:
Building the Swift Driver for the compiler instead of the C++ Driver was just enabled last month in #37122. I think it requires having a whole Swift toolchain for the platform already installed, though I haven't tried it.

I'm just skipping it for now because I think it requires additional instructions on installing a host Swift toolchain and I haven't tried it, but we should enable it at some point.

Reply:
Alright, I just tried it again without that flag. With a Swift toolchain installed in `/usr/bin` in my Fedora Core 33 VPS, it built fine. If I removed that toolchain package, it spits out this warning and continues to build cmark:
```
-- Warning: {} Host toolchain could not locate a compiler to build swift-driver. (Try `--skip-early-swift-driver`)
```
I think it used to fail before, which is why I started using that flag.

I think it's better not to confuse noobs with that warning, but it appears to still work when the swift driver is not built early, so we could remove that flag now.

Reply:
Makes sense. Given the warning, I agree it's better to have the `--skip` flag.